Mathlib/Analysis/InnerProductSpace/Spectrum.lean

```diff
@@ -108,6 +108,31 @@ theorem orthogonalComplement_iSup_eigenspaces (hT : T.IsSymmetric) (μ : 𝕜) :
   have H₂ : eigenspace T μ ⟂ p := (Submodule.isOrtho_orthogonal_right _).mono_left (le_iSup _ _)
   exact H₂.disjoint
 
+/-- Every generalized eigenspace of a symmetric operator is an eigenspace. -/
+theorem genEigenspace_eq_eigenspace
+    {T : E →ₗ[𝕜] E} (hT : T.IsSymmetric) {n : ℕ} {μ : 𝕜} (hn : 1 ≤ n) :
+    genEigenspace T μ n = genEigenspace T μ 1 := by
```

Comment:
I think this and the `maxGenEigenspace` result should go immediately after `conj_eigenvalue_eq_self`